README.md

```diff
@@ -93,6 +94,12 @@ The easiest way to install Iris is with [conda](https://conda.io/miniconda.html)
 Detailed instructions, including information on installing from source,
 are available in [INSTALL](INSTALL).
 
+# Get in touch
+
+  * Report bugs, or suggest new features using an Issue or Pull Request on [Github](https://github.com/SciTools/iris). You can also comment on existing Issues and Pull Requests.
+  * For discussions from a user perspective you could join our [SciTools User's Google Group](https://groups.google.com/forum/#!forum/scitools-iris).
+  * For those involved in developing Iris we also have a [Google Group](https://groups.google.com/forum/#!forum/scitools-iris-dev).
```

Comment:
Can you replace the link text "Google Group" with something more specific, like "Iris Developers Group" ?
Otherwise all looks good to me.  Positive progress !

Reply:
@pp-mo Will do...

Reply:
@pp-mo I also removed a rogue apostrophe.  Although this name change does mean that there is now an 'a' which wants to be an 'an' (L.101).  Shall I fix that?

Reply:
I'll fix it.

Reply:
Oh no! That's still wrong, hang on, I'm doing a lot of context switching so far today, I'll get this right in a minute...

Reply:
Oh no that is right after all.  Don't panic, I've obviously just got my Monday brain in.